spec/services/remove_document_service_spec.rb

```diff
@@ -78,13 +79,40 @@
             explanation: "explanation",
             locale: edition.locale,
             type: "gone",
+            unpublished_at: removal.removed_at,
           },
         )
         described_class.call(edition, removal)
         expect(request).to have_been_requested
       end
     end
 
+    context "when a removal already exists" do
+      it "maintains the existing removed_at time" do
+        existing_removed_at = 2.days.ago
```

Comment:
There's a good chance this could end up flaky due to the annoyance of date precisions at Ruby and DB level. You can probably resolve this by adding a .noon on the end.

Reply:
Done 👍